cluster/addons/fluentd-gcp/fluentd-gcp-ds.yaml

```diff
@@ -42,10 +42,10 @@ spec:
           value: --no-supervisor -q
         resources:
           limits:
-            memory: 300Mi
+            memory: 300Mi # fluentd-gcp-memory-limit
```

Comment:
I'd add MAY BE OVERRIDEN at the end, but waiting for @mikedanese suggestion on which way we want to go (# comments vs. {{ params }})